.github/workflows/Tests.yml

```diff
@@ -351,18 +257,18 @@ jobs:
     timeout-minutes: 10
 
     env:
-      PYTHON_VERSION: '3.10'
+      PYTHON_VERSION: '3.12'
       LANG: "en_US.utf-8"
 
     steps:
     - uses: actions/checkout@v4
     - name: Set up Python
       uses: actions/setup-python@v5
       with:
-        python-version: "3.10"
+        python-version: "3.12"
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.